js_modules/dagster-ui/packages/ui-core/src/assets/AssetCatalogAssetGraph.tsx

```diff
@@ -57,12 +58,17 @@ export const AssetCatalogAssetGraph = React.memo(
       },
     );
 
+    const {favorites, loading: favoritesLoading} = useFavoriteAssets();
+
     const fetchOptions = React.useMemo(
       () => ({
-        loading: false,
+        loading: favoritesLoading,
         hideEdgesToNodesOutsideQuery,
+        hideNodesMatching: favorites
+          ? (node: {assetKey: {path: string[]}}) => !favorites.has(tokenForAssetKey(node.assetKey))
+          : undefined,
       }),
-      [hideEdgesToNodesOutsideQuery],
+      [hideEdgesToNodesOutsideQuery, favorites, favoritesLoading],
```

Comment:
Add support for favorites in the catalog asset graph similar to https://github.com/dagster-io/dagster/pull/29317